docs/usage/files.md

```diff
@@ -36,10 +39,14 @@ file's contents, upload new versions, and perform other common file operations
 - [Get File Representations](#get-file-representations)
 - [Get Thumbnail](#get-thumbnail)
 - [Add Metadata](#add-metadata)
+- [Set Metadata](#set-metadata)
 - [Get Metadata](#get-metadata)
 - [Update Metadata Values](#update-metadata-values)
 - [Remove Metadata](#remove-metadata)
 - [Get All Metadata](#get-all-metadata)
+- [Set a Classification](#set-a-classification)
```

Comment:
Speaking of these methods, you might be able to update the implementation of "Set Classification" to use the new set metadata method now